opentracing-reactive-grpc/src/test/java/io/quarkus/qe/GrpcPingPongResourceTest.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.qe;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Disabled;
+
+import io.quarkus.grpc.GrpcService;
+import io.quarkus.qe.pong.GrpcPongService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+
+@Disabled("Caused by https://github.com/quarkusio/quarkus/issues/13224")
+@QuarkusScenario
+@DisabledOnNative
+public class GrpcPingPongResourceTest extends AbstractPingPongResourceIT {
+
+    @Inject
+    @GrpcService
+    GrpcPongService pongService;
```

Comment:
This is not working using `@QuarkusScenario`, the workaround is to expose the last trace ID via a REST endpoint.

Reply:
Right, I've updated the test. Can you double check if this is suitable now?